realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,95 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task assembleRobolectric(description: "Build the Robolectric shared library for Darwin") {
```

Comment:
This task is too big and does too many things. It should be spitted in its components instead